### DISCUSSION TITLE
Vía - Ultralytics YOLOv8 Docs

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#7841] [Docs]
# Vía - Ultralytics YOLOv8 Docs

Aprende a utilizar Ultralytics YOLO  para el seguimiento de objetos en secuencias de vídeo. Guías para utilizar diferentes rastreadores y personalizar las configuraciones de los rastreadores.

https://docs.ultralytics.com/es/modes/track/

**Comment by alexbarnadas** [comment]
Is there not a table of attributes and functions of the tracker class? Like in the Predict class!

**Reply by pderrenger** [reply]
@alexbarnadas actualmente, la documentación de Ultralytics YOLOv8 no proporciona una tabla de atributos y funciones para la clase de rastreador similar a la que existe para la clase de predicción. Sin embargo, la funcionalidad de seguimiento se maneja a través de la API de YOLO y los archivos de configuración YAML específicos del rastreador, como `botsort.yaml` o `bytetrack.yaml`.

Para el seguimiento, generalmente se cargará un modelo y se llamará al método `track()` con los parámetros deseados. Aquí hay un ejemplo de cómo se puede hacer esto:

```python
from ultralytics import YOLO

# Cargar un modelo oficial o personalizado
model = YOLO('yolov8n.pt')  # Cargar un modelo oficial de Detección
# ...

# Realizar seguimiento con el modelo
results = model.track(source="video.mp4", show=True, tracker="bytetrack.yaml")
```

Los resultados del seguimiento se devolverán en un objeto `Results`, que contiene atributos como `boxes` para las cajas delimitadoras, `masks` para las máscaras de segmentación (si están disponibles), y `keypoints` para los puntos clave (si están disponibles).

Si necesitas más detalles sobre los atributos y métodos disponibles para el seguimiento, te recomendaría revisar la documentación oficial de Ultralytics o el código fuente del repositorio de YOLOv8, ya que la documentación y las características pueden actualizarse con el tiempo.
